android/assets/jsons/translations/template.properties

```diff
@@ -882,3 +885,2700 @@ Invalid ID! =
 [stats] from every [param] = 
 [stats] from [param] tiles in this city = 
 +[amount]% vs [unitType] = 
+
```

Comment:
Again it seems to have added the auto-generated strings in the template file - this should not be happening :/

Reply:
It didnt auto-generate. I tried to mess with it using my own scripts. I need to learn how exactly auto-generation around those files work